core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -258,7 +259,7 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                    ng-if="explorationStats[exploration.id].open_feedback.length !==0"
                    ng-show="activeSubDropdown === EXPLORATION_DROPDOWN_STATS.OPEN_FEEDBACK || checkMobileView()">
                 <div ng-repeat="feedback in explorationStats[exploration.id].open_feedback">
-                  <[feedback.text | truncate: 100]> (by <a ng-href="/create/<[exploration.id]>#/feedback"><[feedback.author_username || 'a learner']></a>)
+                  <[feedback.text | truncate: 100]> (by <a ng-href="/create/<[exploration.id]>#/feedback" style="color: #009688;"><[feedback.author_username || 'a learner']></a>)
```

Comment:
Yes since I had named the class feedback, I was reluctant to give this element that class. But after reading your suggestion below, I guess I could just change the class name and apply that class to this.